gadgets/trace_open.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __OPENSNOOP_H
+#define __OPENSNOOP_H
```

Comment:
Since the header files are not used in userspace, should they be moved into the .c files?

Reply:
Yeah, I agree.